### ISSUE TITLE
"[Vue warn]: Failed to resolve component: va-config"

### ISSUE THREAD

**Issue by thunfisch987** [epicmaxco/vuestic-ui#3007]
**Vuestic-ui version:** 1.5.3
**Vuestic/nuxt version:** 1.0.10

**Steps to reproduce**
I don't know what caused it, it randomly appeared, and can't trace it back.

**What is the expected behavior?**
No errors in the Console

**What is the current behavior?**
```
[Vue warn]: Failed to resolve component: va-config
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.
```

**Other information**
Nuxt 3.2.0

**Comment by thunfisch987** [comment]
It seems to be something with my Sidebar, im testing further

**Comment by thunfisch987** [comment]
even just using
```
<VaSidebar>
  <VaSidebarItem>
    <VaSidebarItemContent>
      <VaSidebarItemTitle>Hi</VaSidebarItemTitle>
    </VaSidebarItemContent>
  </VaSidebarItem>
</VaSidebar>
```

**Comment by thunfisch987** [comment]
it makes no difference to have something configured in nuxt.config.ts

my other installed modules are
```
modules: [
		"@vuestic/nuxt",
		"@nuxtjs/device",
		"nuxt-icon",
		"nuxt-typed-router",
		"@nuxt/image-edge",
	],
```

**Comment by thunfisch987** [comment]
To clarify: i've made a sidebar and used it without problems, but this error randomly popped up

**Comment by m0ksem** [comment]
It's related to https://github.com/epicmaxco/vuestic-ui/issues/2637. Will be fixed in 1.6.0 (in next week). Until then, you can try to use [1.5.4-next-8872b22a5-20230213](https://www.npmjs.com/package/vuestic-ui/v/1.5.4-next-8872b22a5-20230213) if it is an app breaker for you.
